### PR TITLE
Make the rubber band color configurable

### PR DESCRIPTION
**Pull request by sbrunner**
Add missing rubber band update
Remove unneeded imports

Fix: https://hub.qgis.org/issues/13793

These features were funded by: MEDDE (French Ministry of Sustainable Development)

### CODE REVIEW

**Comment by m-kuhn** [comment]
Does this only affect the node tool?

**Comment by sbrunner** [comment]
Yes :-)

**Comment by m-kuhn** [comment]
Would it make sense to make it more generic?
Is it apparent from the change in the options that it only affects the node tool? (I have only read the text "Select color" and I'm not sure if it is used in a context that makes it obvious)

**Comment by nyalldawson** [comment]
I agree - it would be nice if the same setting could be used to customise the digitizing new feature rubber band too. It's a common complaint that the digitizing rubber band is the wrong color/transparency.

**Comment by sbrunner** [comment]
OK, thanks for your feed-backs I will have a look :-)

**Comment by sbrunner** [comment]
Pull request updated :-)

**Comment by m-kuhn** [comment]
Is there a reason that the (new) button for the fill color is not next to the line color?

**Comment by sbrunner** [comment]
No special reason, you have a better purpose?

**Comment by m-kuhn** [comment]
I'd have expected the fill color to be part of the rubberband and therefore in the group box with the title rubberband.

**Comment by sbrunner** [comment]
sorry I miss the negation of your first message,

for me the  fill color is  next to the line color not for you?

**Comment by sbrunner** [comment]
qgis

**Comment by m-kuhn** [comment]
It landed in the feature creation box. Maybe caused by a rebase on master?

**Comment by m-kuhn** [comment]
When I open the un-rebased commit, I see it at the right position.
Can you rebase and make sure it's where you want?

**Comment by sbrunner** [comment]
On my rebase it's working well, I just push the rebase :-)

**Comment by m-kuhn** [comment]
Merged in dad7aa70f62bf35b

Can you check if it's all ok, I had to manually move the field, it somehow landed in the topmost groupbox again...
Thank you.

**Comment by sbrunner** [comment]
Thanks @m-kuhn

File: src/app/nodetool/qgsmaptoolnodetool.cpp [hunk 1]
```diff
@@ -360,7 +360,16 @@ void QgsMapToolNodeTool::updateSelectFeature()
 
   mSelectRubberBand = new QgsGeometryRubberBand( mCanvas, mSelectedFeature->geometry()->type() );
   mSelectRubberBand->setBrushStyle( Qt::SolidPattern );
-  mSelectRubberBand->setFillColor( QColor( 255, 0, 0, 50 ) );
+
+  QSettings settings;
+  QColor color(
+    settings.value( "/qgis/digitizing/select_color_red", 255 ).toInt(),
```
Comment (dgoedkoop) [inline src/app/nodetool/qgsmaptoolnodetool.cpp]: Is there a reason why "select_color_red/green/blue" is used here, while in the options screen, only "line_color_red/green/blue" can be configured?

Comment (sbrunner) [inline src/app/nodetool/qgsmaptoolnodetool.cpp]: Effectively @dgoedkoop , I just open a pull request here to fix that: #2608

**Pull request closed without merging**